src/test/java/unit/SshKey.java

```diff
@@ -0,0 +1,51 @@
+package unit;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import bio.terra.cli.serialization.userfacing.UFSshKey;
+import harness.TestCommand;
+import harness.TestUser;
+import harness.baseclasses.ClearContextUnit;
+import java.io.IOException;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+@Tag("unit")
+public class SshKey extends ClearContextUnit {
+
+  TestUser testUser = TestUser.chooseTestUser();
```

Comment:
You can put the initialization of `testUser` and the login in a `@BeforeEach` method if you like.